geant4.sh

```diff
@@ -40,11 +40,30 @@ cmake $SOURCEDIR                                    \
   -DCMAKE_STATIC_LIBRARY_C_FLAGS="-fPIC"            \
   -DGEANT4_USE_G3TOG4=ON                            \
   -DGEANT4_INSTALL_DATA=ON                          \
-  -DGEANT4_USE_SYSTEM_EXPAT=OFF
+  -DGEANT4_USE_SYSTEM_EXPAT=OFF                     \
+  ${XERCESC_ROOT:+-DGEANT4_USE_OPENGL_X11=ON -DGEANT4_USE_GDML=ON -DXERCESC_ROOT_DIR=$XERCESC_ROOT}
 
 make ${JOBS+-j $JOBS}
 make install
 
+#Get data file versions:
+. $INSTALLROOT/bin/geant4.sh
```

Comment:
Can you use `source` please. We use `bash`, no need to be POSIX-compliant and this is consistent with most of the other recipes.